.github/workflows/containers.yml

```diff
@@ -17,22 +17,28 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        container: [ functional, integration, unit ]
+        include:
+          - directory: docker/network
+            prefix: ovirt/vdsm-network-tests
+            container: [ functional, integration, unit ]
+          - directory: docker
+            prefix: ovirt/vdsm
+            container: [ test ]
```

Comment:
OK. One way or another, the `Makefiles` (one or both) need to be modified to match the target used in this workflow.
Since we do:
```
run: make ${{ matrix.container }}
...
image: ${{ matrix.prefix }}-${{ matrix.container }}
```
The make target is part of the (or the entire) name of the container image. I'll try a different way to unify.

Reply:
Network makefiles already take `centos-8` and `centos-9` as targets, but in the Action they are uploaded as full containers with multiple tags. Therefore, we do not want the container tag to be `container: [ centos-8, centos-9 ]`, because that would just build them separately. We want all distros (tags) to be built under the same makefile target.

There is no easy way to keep the same name for containers, both for general testing and for network, without adding a target to the docker makefile, as is done currently in this branch. This way we can simply use the same strategy for all containers and have the `vdsm-test:<tag>` containers uploaded.